### DISCUSSION TITLE
Welcome to argent-x Discussions!

### DISCUSSION THREAD

**Discussion by juniset** [argentlabs/argent-x#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by uf96800** [comment]
I think some areas need to be improved, the back or back button is not enough, for example, after I click A1 in the upper left corner, there is no way to return to the original wallet.

**Comment by delsqrdpsi** [comment]
Can someone spot me to what are the list of all the **selector functions** that I can use for session keys policies:
     policies: [
    {
      contractAddress: "0x...",
      selector: **"doAction"**
    }
  ]
